pkg/sql/copy.go

```diff
@@ -424,6 +424,17 @@ func (c *copyMachine) readCSVTuple(ctx context.Context, record []string) error {
 			exprs[i] = tree.DNull
 			continue
 		}
+		switch t := c.resultColumns[i].Typ; t.Family() {
+		case types.BytesFamily,
```

Comment:
where did you get this list from? how do we know it's complete?
(it's weird to me that e.g. Timestamp is ehre but Time is not)

Reply:
the list comes from from what `COPY FROM ... TEXT` is already doing: https://github.com/cockroachdb/cockroach/blob/61fdb08be3835e3c4e18ba6c0389bc7edc43d7d1/pkg/sql/copy.go#L660-L670

it seems old, wouldn't be surprised if it has the wrong types in it

i can extract that into its own function

Reply:
hmmm actually looks like the list we have is already too big -- PG doesn't do this decoding for `TEXT` columns. i think we only want this for `BytesFamily`

Reply:
hmmm actually this is wrong. I tested in PG, and COPY CSV does not do `\xdd` or `\ddd` escaping. Left a comment on https://github.com/cockroachdb/cockroach/issues/68804 explaining why that issue is happening